R/dependencies.R

```diff
@@ -163,16 +168,17 @@ dirDependenciesRenv <- function(dir) {
     on.exit(Sys.setenv(RENV_PROFILE = profile), add = TRUE)
   }
 
+  absDir <- normalizePath(dir)
```

Comment:
I think you'll want `winslash = "/"` here, since `renv` always tries to parse paths using forward slashes. (Or, the path / slash normalization should happen in `ignoresForRenv`.)